etc/packages.yml

```
@@ -24,9 +24,20 @@ package:
 
   # elastic/integrations
   registry_data:
-    # integration package schema version
-    format_version: "1.0.0"
+    categories: ["security"]
     conditions:
       kibana_version: "^7.13.0"
-    # this determines the version for the package-storage generated artifact
-    version: "0.0.1-dev.3"
+    description: "Prebuilt detection rules for Elastic Security"
+    format_version: "1.0.0"
+    icons:
+    - src: "/img/security-logo-color-64px.svg"
+      size: "16x16"
+      type: "image/svg+xml"
+    license: basic
```

internal is currently being set to `null`

```suggestion
    internal: true
    license: basic
```

---

that's a good thing. it's not supposed to be internal anymore